pandas/tests/groupby/test_groupby.py

```diff
@@ -2509,3 +2509,53 @@ def test_rolling_wrong_param_min_period():
     result_error_msg = r"__init__\(\) got an unexpected keyword argument 'min_period'"
     with pytest.raises(TypeError, match=result_error_msg):
         test_df.groupby("name")["val"].rolling(window=2, min_period=1).sum()
+
+
+@pytest.mark.parametrize(
```

Comment:
move these to pandas/tests/groupby/aggregate/test_aggreagte.py (or maybe test_cython) see where similar are.